examples/webgl_pmrem_test.html

```diff
@@ -0,0 +1,203 @@
+                renderer.setSize( width, height );
+                renderer.gammaOutput = true;
+                renderer.physicallyCorrectLights = true;
+
+                // tonemapping
+                renderer.toneMapping = THREE.ACESFilmicToneMapping;
+                renderer.toneMappingExposure = 1;
+
+                document.body.appendChild( renderer.domElement );
+
+                window.addEventListener( 'resize', onResize, false );
+
+                // scene
+
+                scene = new THREE.Scene();
+
+                // camera
+
+                camera = new THREE.PerspectiveCamera( 40, aspect, 1, 30 );
+                updateCamera();
+                camera.position.set( 0, 0, 16 );
+
+                // controls
+
+                controls = new OrbitControls( camera, renderer.domElement );
+                controls.update();
+
+                // light
+
+                var directionalLight = new THREE.DirectionalLight( 0xffffff, 1 );
+                var x = 597;
+                var y = 213;
+                var theta = ( x + 0.5 ) * Math.PI / 512;
+                var phi = ( y + 0.5 ) * Math.PI / 512;
+
+                directionalLight.position.setFromSphericalCoords( 100, - phi, Math.PI/2 - theta );
+
+                scene.add( directionalLight );
+                // scene.add( new THREE.DirectionalLightHelper( directionalLight ) );
+
+                document.body.addEventListener( 'mouseover', function () {
+
+                    scene.traverse( function ( child ) {
+
+                        if ( child.isMesh ) {
+
+                            child.material.envMapIntensity = 1;
+                            directionalLight.intensity = 0;
+                        
+                        }
+
+                    } );
+
+                } );
+
+                document.body.addEventListener( 'mouseout', function () {
+
+                    scene.traverse( function ( child ) {
+
+                        if ( child.isMesh ) {
+
+                            child.material.envMapIntensity = 0;
+                            directionalLight.intensity = 1;
+
+                        }
+
+                    } );
+
+                } );
+
+			}
+
+			function createObjects() {
+
+                var radianceMap = null;
+                new RGBELoader()
+                    .setDataType( THREE.UnsignedByteType )
+                    // .setDataType( THREE.FloatType )
+                    .setPath( 'textures/equirectangular/' )
+                    .load( 'spot1Lux.hdr', function ( texture ) {
+
+                        // This HDR environment map is expressed in nits (lux / sr). The directional light has units of lux,
+                        // so to match a 1 lux light, we set a single pixel with a value equal to 1 divided by the solid 
+                        // angle of the pixel in steradians. This image is 1024 x 512, 
+                        // so the value is 1 / (pi / 512) ^ 2 = 26,560 nits.
```

Comment:
I would have expected an adjustment based on the sine of the polar angle. Pixles near the poles map to less area.

Reply:
Good call. Of course it's likely washed out by the interpolation from equirectangular to cubemap, but I've updated the comment.